runtime/src/iree-amd-aie/aie_runtime/iree_aie_runtime.h

```diff
@@ -536,6 +536,10 @@ struct AMDAIEDeviceModel {
 
   FailureOr<std::array<uint32_t, 3>> getAIEMatmulInstructionSize(
       Type elTypeLhs, Type elTypeRhs, Type elTypeAcc) const;
+
```

Comment:
Where did you find these values?

Reply:
From mlir-aie: https://github.com/Xilinx/mlir-aie/blob/043267ec235b1d2a1dc957ff2e786a535c52c0ce/include/aie/Dialect/AIE/IR/AIETargetModel.h#L399